### PR TITLE
Fix unresolved column name when using table.column

### LINKED ISSUE #511: Cannot resolve column name when specifying table.column
For example: 
`select t.a from t`

### PR DESCRIPTION
**Pull request by birdstorm**
Fix #511, the root cause is missing subquery alias when resolving `UnresolvedRelations`

### CODE REVIEW

File: tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java [hunk 1]
```diff
@@ -105,8 +105,7 @@ public int getValue() {
   // lockResolverClient, after implements the
   // write implementation of tispark, we can change
   // it to private
-  @VisibleForTesting
-  public final LockResolverClient lockResolverClient;
+  @VisibleForTesting public final LockResolverClient lockResolverClient;
```
Comment (zhexuany) [inline tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java]: please undo this change

**Comment by ilovesoup** [comment]
Please give more description of your change to ease your reviewer :) @birdstorm

File: tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java (hunk 1 above)
Comment (birdstorm) [reply tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java]: Why，it is formatted this way.

Comment (birdstorm) [reply tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java]: When you add an annotation before a variable/constant, google format state that they should be in the same line.

**Comment by birdstorm** [comment]
@ilovesoup done

File: tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java (hunk 1 above)
Comment (zhexuany) [reply tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java]: Interesting - -

**Pull request merged**